lhotse/supervision.py

```diff
@@ -512,6 +513,89 @@ def from_dicts(data: Iterable[Dict]) -> "SupervisionSet":
+
+        - ``Type``  --  segment type; should always by ``SPEAKER``
+        - ``File ID``  --  file name; basename of the recording minus extension (e.g.,
+        ``rec1_a``)
+        - ``Channel ID``  --  channel (1-indexed) that turn is on; should always be
+        ``1``
+        - ``Turn Onset``  --  onset of turn in seconds from beginning of recording
+        - ``Turn Duration``  -- duration of turn in seconds
+        - ``Orthography Field`` --  should always by ``<NA>``
+        - ``Speaker Type``  --  should always be ``<NA>``
+        - ``Speaker Name``  --  name of speaker of turn; should be unique within scope
+        of each file
+        - ``Confidence Score``  --  system confidence (probability) that information
+        is correct; should always be ``<NA>``
+        - ``Signal Lookahead Time``  --  should always be ``<NA>``
+
+        For instance:
+
+            SPEAKER CMU_20020319-1400_d01_NONE 1 130.430000 2.350 <NA> <NA> juliet <NA> <NA>
+            SPEAKER CMU_20020319-1400_d01_NONE 1 157.610000 3.060 <NA> <NA> tbc <NA> <NA>
+            SPEAKER CMU_20020319-1400_d01_NONE 1 130.490000 0.450 <NA> <NA> chek <NA> <NA>
+
+        :param path: Path to RTTM file.
+        :return: a new ``SupervisionSet`` instance containing segments from the RTTM file.
+        """
+        segments = []
+        with open(path, "r") as f:
+            for idx, line in enumerate(f):
+                parts = line.strip().split()
+                assert len(parts) == 10, f"Invalid RTTM line: {line}"
+                recording_id = parts[1]
+                segments.append(
+                    SupervisionSegment(
+                        id=f"{recording_id}-{idx:06d}",
+                        recording_id=recording_id,
+                        channel=int(parts[2]),
+                        start=float(parts[3]),
+                        duration=float(parts[4]),
+                        speaker=parts[7],
+                    )
+                )
+        return SupervisionSet.from_segments(segments)
+
+    @staticmethod
+    def from_rttms(paths: Union[Pathlike, Iterable[Pathlike]]) -> "SupervisionSet":
+        """
+        Read multiple RTTM files located at ``paths`` and create a :class:`.SupervisionSet` manifest for them.
+
+        .. code:: python
+
+            >>> from lhotse import SupervisionSet
+            >>> sup1 = SupervisionSet.from_rttms(Path('/path/to/rttm_dir'))
+            >>> sup2 = SupervisionSet.from_rttms(Path('/path/to/rttm_dir').rglob('ref_*'))
+
+        .. hint::
+
+            For creating a SupervisionSet from a single RTTM file, use the :meth:`.from_rttm` method.
+
+        :param paths: Path to RTTM files (can be path to a directory or an iterable of paths to RTTM files).
+        :return: a new ``SupervisionSet`` instance containing segments from the RTTM files.
+        """
+        if isinstance(paths, Pathlike.__args__):
+            paths = check_and_rglob(paths, "*.rttm")
```

Comment:
I think this rglob pattern wouldn't find files named `rttm` which are somewhat common in Kaldi recipes?

Reply:
I suppose in that case I will remove rglob and just assume that the user has provided the path to a file or an iterable of paths.